docs/src/api.md

```diff
@@ -68,30 +72,52 @@ The `search_query` is not included in the error response if parsing the search q
 
 ## Export to PDF
 
-If the PDF report needs to be downloaded programmatically, e.g. for inclusion in a release package, use the endpoint: `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf`. No authorization is needed for this endpoint.
+If the PDF report needs to be downloaded programmatically, e.g. for inclusion in a release package,
+use the endpoint: `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf`.
+No authorization is needed for this endpoint.
 
-The `report_uuid` is the unique identifier that *Quality-time* assigns to a report. It can be found by navigating to a report in the browser and looking for the `report_uuid` in the address bar. For example, when the URL in the browser's address bar is `https://www.quality-time.example.org/f1d0e056-2440-43bd-b640-f6753ccf4496?hidden_columns=comment`, the part between the last slash and the question mark is the `report_uuid`.
+The `report_uuid` is the unique identifier that *Quality-time* assigns to a report.
+It can be found by navigating to a report in the browser and looking for the `report_uuid` in the address bar.
+For example, when the URL in the browser's address bar is `https://www.quality-time.example.org/f1d0e056-2440-43bd-b640-f6753ccf4496?hidden_columns=comment`,
+the part between the last slash and the question mark is the `report_uuid`.
 
-To hide metrics that do not need any action, set the `metrics_to_hide` parameter to `no_action_needed`, i.e. `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf?metrics_to_hide=no_action_needed`. To hide all metrics and only export the report dashboard to PDF, set the `metrics_to_hide` parameter to `all`.
+To hide metrics that do not need any action, set the `metrics_to_hide` parameter to `no_action_needed`,
+i.e. `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf?metrics_to_hide=no_action_needed`.
+To hide all metrics and only export the report dashboard to PDF, set the `metrics_to_hide` parameter to `all`.
 
-To hide columns from the report, set the `hidden_columns` parameter, for example `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf?hidden_columns=target,comment`. Possible options are `trend`, `status`, `measurement`, `target`, `source`, `comment`, `issues`, and `tags`.
+To hide columns from the report, set the `hidden_columns` parameter, for example `https://www.quality-time.example.org/api/v3/report/<report_uuid>/pdf?hidden_columns=target,comment`.
+Possible options are `trend`, `status`, `measurement`, `target`, `source`, `comment`, `issues`, and `tags`.
```

Comment:
These values are not up-to-date. At least `delta` is missing (my fault). 

```suggestion
Possible options are `trend`, `status`, `measurement`, `delta`, `target`, `source`, `comment`, `issues`, and `tags`.
```

Reply:
Hmm, I'm not sure how to get an exhaustive list. Searching on `hidden_columns` only yields test files, so I used `hiddenColumns.includes`. Is this complete? It seems that the value `unit` does not have a test case

Reply:
To be addressed in #8237.